src/Storage/RecordModifier.php

```diff
@@ -109,16 +109,16 @@ private function setPostedValues(Content $content, $formValues, $contentType)
         $formValues = Input::cleanPostedData($formValues);
         unset($formValues['contenttype']);
 
+        $user = $this->app['users']->getCurrentUser();
         if ($id = $content->getId()) {
             // Owner is set explicitly, is current user is allowed to do this?
             if (isset($formValues['ownerid']) && (integer) $formValues['ownerid'] !== $content->getOwnerid()) {
-                if (!$this->app['permissions']->isAllowed("contenttype:{$contentType['slug']}:change-ownership:$id")) {
```

Comment:
During the controller refactor we decided that Bolt\Users needed to be free of such things…